lib/katello/tasks/delete_docker_v1_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  task :delete_docker_v1_content do
+    docker_repositories = Katello::Repository.where(:content_type => 'docker')
+
+    v1_repos = docker_repositories.select do |repo|
+      Katello.pulp_server.extensions.repository.docker_images(repo.pulp_id).map { |image| image['_ns'] }.uniq == ["units_docker_image"]
```

Comment:
@komidore64 it looks like the `docker_images` method does the correct filtering already:

```ruby
      # Retrieves the docker images for a single repository
      #
      # @param  [String]                id the ID of the repository
      # @return [RestClient::Response]  the set of repository docker images
      def docker_images(id)
        criteria = {:type_ids => [Runcible::Extensions::DockerImage.content_type]}
        unit_search(id, criteria).map { |i| i['metadata'].with_indifferent_access }
      end
```

and `DockerImage.content_type` is just "docker_image"